### PR TITLE
Add a note about custom tabs support when targeting Android API 30 or higher

### PR DESCRIPTION
**Pull request by eth-ellis**
Added the custom tabs support note from
https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/communication/authentication?tabs=android#get-started
to
https://learn.microsoft.com/en-us/azure/developer/mobile-apps/azure-mobile-apps/quickstarts/maui/authentication?pivots=vs2022-windows#configure-the-android-app-for-authentication

### CODE REVIEW

**Comment by Court72** [comment]
@adrianhall

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: articles/mobile-apps/azure-mobile-apps/quickstarts/maui/authentication.md [hunk 1]
```diff
@@ -182,6 +182,18 @@ namespace TodoApp.MAUI
...
+In the _Platforms/Android/AndroidManifest.xml_ file, add the following `queries/intent` nodes in the `manifest` node:
+
+```xml
+<queries>
+  <intent>
+    <action android:name="android.support.customtabs.action.CustomTabsService" />
+  </intent>
+</queries>
+```
+
```
Comment (adrianhall) [inline articles/mobile-apps/azure-mobile-apps/quickstarts/maui/authentication.md:196]: This is authentication - is there an MSAL reference that we can also specifically mention?  Closest I could come up with is this: https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-net-xamarin-android-considerations

**Review by adrianhall — CHANGES_REQUESTED** [review]
Let's finish the conversation and adjustments discussed in the review.

**Comment by adrianhall** [comment]
Since OP has gone silent, let's accept this one and I'll make the recommended adjustment separately.

#sign-off

**Review by adrianhall — APPROVED** [review]

**Comment by adrianhall** [comment]
... I've included these changes in a large PR around authentication.  Please close this PR.

#please-close

**Pull request closed without merging**